ccmlib/scylla_cluster.py

```diff
@@ -42,6 +42,8 @@ def __init__(self, path, name, partitioner=None, install_dir=None,
 
         self.started = False
         self.force_wait_for_cluster_start = (force_wait_for_cluster_start != False)
+        self.default_wait_other_notice_timeout = 120 if self.scylla_mode != 'debug' else 600
```

Comment:
120 isn't enough apparently, I've seen it timeout locally in dev mode and also on jenkins in release mode

Reply:
we seen it only on 50 nodes case ? i.e. this change is for that.

I don't want to double the default time, just for 1 test case

Reply:
https://jenkins.scylladb.com/view/master/job/scylla-master/job/dtest-release/314/testReport/update_cluster_layout_tests/TestLargeScaleCluster/Run_Dtest_Parallel_Cloud_Machines___LongDtest___long_split000___test_add_many_nodes_under_load/
```
ccmlib.node.TimeoutError: watch_rest_for_alive() timeout after 120 seconds
```

Why does it matter to increase the default value while timeouts are rare?

Reply:
This happened on node32 node: https://jenkins.scylladb.com/view/master/job/scylla-master/job/dtest-release/314/artifact/logs-long.release.000/1690338660733_update_cluster_layout_tests.py%3A%3ATestLargeScaleCluster%3A%3Atest_add_many_nodes_under_load/node32.log

Reply:
it mean that in cases of failure, we'll wait more time until we get to the failure.
if for release it works for 99% of tests, I'm o.k. with 1% of the test specifying 
that they need extra timeout for them to boot.
 
I can easily put a default 1h, and it would probably would be enough for all of the cases,
devs are complaining again again, on how how slow/hard is to debug a dtest, not me.